example/spec/helpers/apiUtils.js

```diff
@@ -12,6 +14,27 @@ function setDistributionApiEnvVars() {
   }
 }
 
+async function waitForApiRecord(getMethod, params, matchParams, retryConfig = {}) {
```

Comment:
We should consider reconciling the changes here: https://github.com/nasa/cumulus/pull/2276/files#diff-973a64715495ee3781bfc9fbf9f3bea87af1e453bd62714f7cc80863f5edba96R15

Reply:
That said, 👍🏻

Reply:
I'm not sure what we would reconcile. I looked into whether your `waitForApiStatus` method could use `waitForApiRecord` or something, but it seemed infeasible given that status takes a set of possible statuses (so multiple possible values for `status`, whereas `waitForApiRecord` does a deep equality comparison on the `matchParams` to the returned record, so it can't have multiple values for a given property

Reply:
I was thinking we could write a more general helper, but fair enough.

More generally I think we just need to make a point to get this back into the feature branch before someone picks up the granule work.     I've added a comment to that ticket, and that should suffice.